tests/network/BUILD.bazel

```diff
@@ -8,6 +8,7 @@ go_library(
         "framework.go",
         "networkpolicy.go",
         "primary_pod_network.go",
+        "network-probes.go",
```

Comment:
Maybe you could drop the `network` in the name, given that it will be under the `network` subdir? So that the path looks like `tests/network/probes.go`, wdyt?

Reply:
I changed the name to emphasize these are probes tests that related to network,
so when there will be other probes tests it will be clear which file tests a kind of probes.
But like you suggests it could be understood from the path
I will change it

Reply:
Awesome, thanks!